changedetectionio/tests/test_group.py

```diff
@@ -321,3 +324,28 @@ def test_clone_tag_on_quickwatchform_add(client, live_server):
 
     res = client.get(url_for("tags.delete_all"), follow_redirects=True)
     assert b'All tags deleted' in res.data
+
+def task(include_filters: list, include_filters_from_tags: list, i: int, return_dict: dict):
+    return_dict[i] =  merge_filters(include_filters, include_filters_from_tags)
+
+def test_merge_filters_function_unit():
+    include_filters = ['#only-meaningless1', '#only-meaningless2', '#only-this']
+    include_filters_from_tags = ['//body/b[1]', '/html/body/b[2]']
+    try:
+        # when you check by hand.
+        # It won't work in github CICD.
+        multiprocessing.set_start_method('spawn')
+    except RuntimeError:
+        pass
+    manager = multiprocessing.Manager()
+    return_dict = manager.dict()
+    # if spawn works,
+    # from math import fractorial
+    # 1 - (1/(factorial(5)) ** 5)
+    # 0.9999999999598123
+    processes = [Process(target=task, args=(include_filters, include_filters_from_tags, i, return_dict)) for i in range(5)]
+    [process.start() for process in processes]
+    [process.join() for process in processes]
+    res = { '_'.join(i) for i in return_dict.values()}
+    assert len(res) == 1
```

Comment:
In spawn mode, len(res) will bigger than 1.(you can check with your box.`pytest -k  test_merge_filters_function_unit -xsv tests/test_group.py`)
In other mode, it will passed.
But it can't pass the last test with quite a high probability.

Reply:
for spawn one,
![image](https://github.com/dgtlmoon/changedetection.io/assets/61624808/e30d4c6b-c902-4835-96ea-cc99344f94ee)
for fork one,
![image](https://github.com/dgtlmoon/changedetection.io/assets/61624808/42030379-ab6e-4d54-b233-2741ed2f4399)